plugins/defaultplugins/aclplugin/vppcalls/acl_vppcalls.go

```diff
@@ -23,25 +23,35 @@ import (
 	"git.fd.io/govpp.git/api"
 	"github.com/ligato/cn-infra/logging"
 	"github.com/ligato/cn-infra/logging/logrus"
-	acl_api "github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/bin_api/acl"
+	"github.com/ligato/cn-infra/logging/timer"
+	aclApi "github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/bin_api/acl"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/model/acl"
+	"time"
 )
 
 // AddIPAcl create new L3/4 ACL. Input index == 0xffffffff, VPP provides index in reply.
-func AddIPAcl(rules []*acl.AccessLists_Acl_Rule, aclName string, log logging.Logger, vppChannel *api.Channel) (uint32, error) {
+func AddIPAcl(rules []*acl.AccessLists_Acl_Rule, aclName string, log logging.Logger, vppChannel *api.Channel, stopwatch *timer.Stopwatch) (uint32, error) {
+	// ACLAddReplace time measurement
+	start := time.Now()
+	defer func() {
+		if stopwatch != nil {
+			stopwatch.LogTimeEntry(aclApi.ACLAddReplace{}, time.Since(start))
+		}
+	}()
+
 	// Prepare Ip rules
 	aclIPRules, err := transformACLIpRules(rules)
 	if err != nil {
 		return 0, err
 	}
 	if len(aclIPRules) != 0 {
-		msg := &acl_api.ACLAddReplace{}
+		msg := &aclApi.ACLAddReplace{}
```

Comment:
was there any golint issue? by using the underscores in import aliases we can easily identify that it is "import alias"